CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## master / unreleased
 
+* [ENHANCEMENT] Add compaction_traces_combined_total metric. [#339](https://github.com/grafana/tempo/pull/339)
```

Comment:
the full name of the metric will be `tempodb_compaction_objects_combined`.  Let's put the full name here to help people who read the changelog find the new metric.